tests/test_connection.py

```diff
@@ -0,0 +1,57 @@
+from tempfile import NamedTemporaryFile
+from textwrap import dedent
+
```

Comment:
What about environment variables in config? Do we still support them? Don't we need tests for it (maybe they are tested in python connector project?)?

If we still support them, then is their syntax still the same? Won't changes in README necessary?

Reply:
Those are supported by snowflek-connector-python implementation. So we basically get this for free.